pataruco/mars-rover
# TypeScript

- Scaffold ts challenge
- Move Robot
- Test movement
- Check is in boundaries
- Draft scent
- Draft scent
- Check if is in boundaries
- Set lost
- Test reports last known coordinates when is lost
- Recording previous position 🎉
- Save lost coordinate
- Report lost coordinate
- Remove redundant test
- Draft not move if is on scent
- Scent detect working
- Move types
- draft world class
- Create setter for lostCoordinates
- Set world dimensions via a setter
- World is working 🌏
- draft input handler
- Draft output
- Make output
- Set shebang
- TS almost done
- Dockerised 🐳
- Update readme
